examples/test_catalog_management_v1_examples.py

```diff
@@ -0,0 +1,1622 @@
+
+            # Load the configuration
+            global config
+            config = read_external_sources(CatalogManagementV1.DEFAULT_SERVICE_NAME)
+
+            global account_id
+            account_id = config['ACCOUNT_ID']
+            assert account_id is not None
+
+            global git_auth_token_for_public_repo
+            git_auth_token_for_public_repo = config['GIT_TOKEN']
+            assert git_auth_token_for_public_repo is not None
+
+            global cluster_id
+            cluster_id = config['CLUSTER_ID']
+            assert cluster_id is not None
+
+            # Get bearer token
+            cls.catalog_management_service.get_catalog_account()
+            authenticator = cls.catalog_management_service.get_authenticator()
+            token_manager = authenticator.token_manager
+            bearer_token = token_manager.request_token()['refresh_token']
+            assert bearer_token is not None
+
+        print('Setup complete.')
+
+    needscredentials = pytest.mark.skipif(
+        not os.path.exists(config_file), reason="External configuration not available, skipping..."
+    )
+
+    @needscredentials
+    def test_create_catalog_example(self):
+        """
+        create_catalog request example
+        """
+        try:
+            print('\ncreate_catalog() result:')
+            global catalog_id
+            # begin-create_catalog
+
+            catalog = self.catalog_management_service.create_catalog(
+                label='Catalog Management Service',
+                tags=['python', 'sdk'],
+                kind='vpe',
+                owning_account=account_id
+            ).get_result()
+
+            print(json.dumps(catalog, indent=2))
+
+            # end-create_catalog
+            assert catalog is not None
+            assert catalog['id'] is not None
+            catalog_id = catalog['id']
+
+        except ApiException as e:
+            pytest.fail(str(e))
+
+    @needscredentials
+    def test_get_catalog_example(self):
+        """
+        get_catalog request example
+        """
+        try:
+            print('\nget_catalog() result:')
+            # begin-get_catalog
+
+            catalog = self.catalog_management_service.get_catalog(
+                catalog_identifier=catalog_id
+            ).get_result()
+
+            print(json.dumps(catalog, indent=2))
+
+            # end-get_catalog
+
+        except ApiException as e:
+            pytest.fail(str(e))
+
+    @needscredentials
+    def test_replace_catalog_example(self):
+        """
+        replace_catalog request example
+        """
+        try:
+            print('\nreplace_catalog() result:')
+            # begin-replace_catalog
+
+            catalog = self.catalog_management_service.replace_catalog(
+                catalog_identifier=catalog_id,
+                id=catalog_id,
+                tags=['python', 'sdk', 'updated'],
+                owning_account=account_id,
+                kind='vpe'
```

Comment:
If it's not a big deal please add trailing commas to every lines like this. Just like in the integration tests.
```suggestion
                kind='vpe',
```